test/imp/noop_imp.js

```diff
@@ -1,33 +1,41 @@
+function NoopSpanContextImp() {
+};
+NoopSpanContextImp.prototype.setBaggageItem = function (key, value) {
+};
+NoopSpanContextImp.prototype.getBaggageItem = function (key) {
+};
+
 function NoopSpanImp() {
-}
+    this._context = new NoopSpanContextImp();
+};
 NoopSpanImp.prototype.tracer = function () {
     return null;
 };
+NoopSpanImp.prototype.context = function () {
+    // XXX what's the deal here? Why can tracer() return null?
```

Comment:
Per the comment before, this can return `null` as the SpanContext has an API wrapper. In either case, this comment should be removed.